worker/containerd/containerd.go

```diff
@@ -32,6 +32,8 @@ func NewWorkerOpt(root string, address, snapshotterName, ns string, rootless boo
 	opts = append(opts, containerd.WithDefaultNamespace(ns))
 	if runtime.GOOS == "freebsd" {
 		opts = append(opts, containerd.WithDefaultRuntime("wtf.sbk.runj.v1"))
+	} else if runtime.GOOS == "darwin" {
+		opts = append(opts, containerd.WithDefaultRuntime("containerd-shim-rund-v1"))
```

Comment:
The default should be probably just empty for now

Reply:
Sure, I just didn't find how to pass runtime explicitly :D

Reply:
The runtime (as well as bindfs helper) should be configured in buildkitd.toml

Reply:
Do I understand it properly that
1. There is currently no way in buildkit to specify containerd runtime for individual build run
2. There is currently no way to specify runtime globally via buildkit config?

Reply:
Currently no way, so it has to be newly implemented.

The config structure can be probably similar to containerd CRI:
https://github.com/containerd/containerd/blob/70a2c95ae8c02d7a4e448f0e4fb8bb0e6344b5c7/pkg/cri/config/config.go#L96

Reply:
Sigh. That'll be a separate PR. I definitely want a global setting. And per-build option could also be useful.

Reply:
Done: https://github.com/moby/buildkit/pull/4141